pkg/cmd/grafana/apiserver/cmd.go

```diff
@@ -85,3 +135,48 @@ func RunCLI() int {
 
 	return cli.Run(cmd)
 }
+
+var _ logr.LogSink = (*logAdapter)(nil)
```

Comment:
does this need to be copied? not sure if this is the issue, but the log adapter could be made public

Reply:
What about https://github.com/grafana/grafana/pull/84530#discussion_r1526166311?

Reply:
I re-used extra options log wrapper by calling `ExtraOptions.Apply(...)`